bundles/org.openhab.binding.openwebnet/README.md

```diff
@@ -208,16 +208,18 @@ OPEN command to execute: *5*8#134##
+| `brightness`                            | `bus_dimmer`, `zb_dimmer`                                     | Dimmer  | To adjust the brightness value (Percent, `ON`, `OFF`)                                                                                                                  |     R/W     |
+| `shutter`                               | `bus_automation`                                              | Rollershutter | To activate roller shutters (`UP`, `DOWN`, `STOP`, Percent - [see Shutter position](#shutter-position))                                                                |     R/W     |
+| `scenario`                              | `bus_scenario_control`                                        | String  | Trigger channel for Basic scenario events [see possible values](#scenario-channels)                                                                                    | R (TRIGGER) |
+| `button#X`                              | `bus_cen_scenario_control`, `bus_cenplus_scenario_control`    | String  | Trigger channel for CEN/CEN+ scenario events [see possible values](#scenario-channels)                                                                                 | R (TRIGGER) |
+| `sensor`                                | `bus_dry_contact_ir`                                          | Switch  | Indicates if a Dry Contact Interface is `ON`/`OFF`, or if an IR Sensor is detecting movement (`ON`), or not  (`OFF`)                                                   |      R      |
+| `power`                                 | `bus_energy_meter`                                            | Number:Power | The current active power usage from Energy Meter                                                                                                                       |      R      |
+| `totalizerDay`                          | `bus_energy_meter`                                            | Number  | Current day energy totalizer                                                                                                                                           |      R      |
+| `totalizerMonth`                        | `bus_energy_meter`                                            | Number  | Current month energy totalizer                                                                                                                                         |      R      |
+| `aux`                                   | `bus_aux`                                                     | String  | Possible commands: `ON`, `OFF`, `TOGGLE`, `STOP`, `UP`, `DOWN`, `ENABLED`, `DISABLED`, `RESET_GEN`, `RESET_BI`, `RESET_TRI`. Only `ON` and `OFF` are supported for now |     R/W     |
```

Comment:
- do not modify lines that are not part of this PR
- channel names must be changed accordingly to new suggested channel names. Copy description from suggested channel definition
- item type should be Number:Energy